integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -35,43 +35,47 @@
   public static final String CREATE_DOMAIN_JOB_MESSAGE =
       "Domain base_domain was created and will be started by the WebLogic Kubernetes Operator";
 
-  private static final Logger logger = Logger.getLogger("OperatorIT", "OperatorIT");
+  protected static final Logger logger = Logger.getLogger("OperatorIT", "OperatorIT");
 
-  private Map<String, Object> domainMap;
-  private Map<String, Object> pvMap;
+  protected Map<String, Object> domainMap;
```

Comment:
you can undo the var changes as you removed the duplicate code you don't need them.

Reply:
I still use domainMap in JRFDomain.java. I reverted all other unused variables back to 'private'.